src/main/java/com/banking/accountservice/repository/MySqlAccountRepository.java

```diff
@@ -0,0 +1,101 @@
+package com.banking.accountservice.repository;
+
+import com.banking.accountservice.dto.Account;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+@Repository
+public class MySqlAccountRepository implements AccountRepository {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @Override
+    public List<Account> findAll() {
+        return jdbcTemplate.query("select * from ACCOUNTS", new RowMapper<Account>() {
+            @Override
+            public Account mapRow(ResultSet rs, int rowNum) throws SQLException {
```

Comment:
As commented in `bank-service`, this method is duplicated in this class 3 times. Code must not be duplicated unless there is a good reason for it.

Reply:
Done at last commit